scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_CreateScheduleAndNofifyScalingEngineTest.java

```diff
@@ -182,7 +184,7 @@ public void testDeleteSchedule() throws Exception {
 
 		// Delete End job.
 		ResultActions resultActions = mockMvc
-				.perform(delete(getSchedulerPath(appId)).accept(MediaType.APPLICATION_JSON));
+				.perform(delete(TestDataSetupHelper.getSchedulerPath(appId, guid)).accept(MediaType.APPLICATION_JSON));
```

Comment:
Same question as policies.test.js in apiserver, MockMvc has param method specifying a query parameter to the request.